README.md

```diff
@@ -8,6 +8,21 @@ It includes options to automatically batch all the `.heic` files it finds in a f
 
 [Download the latest release](https://github.com/danbrakeley/heic2png/releases), and then put the exe somewhere in your path.
 
+### Compiling on Linux
```

Comment:
Technically these same steps should work on any OS, not just Linux. Maybe title this `### Building from source`?